### PR TITLE
Sagemaker Hyperpod Training Adaptor For Nemo Release 1.1.0

### PR DESCRIPTION
**Pull request by rohithn1**
## Description
Adding support for DeepSeek R1 distill Llama and Qwen models for finetuning and Qwen v2 as a prerequisite. 

## Merge Checklist
Put an x in the boxes that apply. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your pull request.

### General
 - [x] I have read the [CONTRIBUTING](../CONTRIBUTING.md) doc
 - [x] I have run `pre-commit run --all-files` on my code. It will check for [this configuration](../.pre-commit-config.yaml).
 - [x] I have updated any necessary documentation, including [READMEs](../README.md) and API docs (if appropriate)
 - [x] I have verified the licenses used in the license-files artifact generated in the Python License Scan CI check. If the license workflow fails, kindly check the licenses used in the artifact.

### Tests
 - [x] I have run `pytest` on my code and all unit tests passed.
 - [x] I have added tests that prove my fix is effective or that my feature works (if appropriate)
 - [x] I have ran a training job with real data (huggingface dataloader instead of synthetic data) and validated that the loss value goes down.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: .github/workflows/license-scan-runner-pull.yml
Comment (haohanchen-aws) [inline .github/workflows/license-scan-runner-pull.yml:1]: Why adding this?

**Review by julianhr — APPROVED** [review]

**Review by haohanchen-aws — APPROVED** [review]

**Review by jessech-en — APPROVED** [review]
